src/pint/toa.py

```diff
@@ -1476,7 +1476,7 @@ def get_groups(self, gap_limit=None):
         # TODO: make all values Quantity objects for consistency
         if gap_limit is None:
             gap_limit = 2 * u.h
-        if "groups" not in self.table or gap_limit != 2 * u.h:
+        if "groups" not in self.table.colnames or gap_limit != 2 * u.h:
```

Comment:
This fixes the `FutureWarning` produced by numpy: `FutureWarning: elementwise == comparison failed and returning scalar instead; this will raise an error or perform elementwise comparison in the future.`
There is some info about this warning here: https://stackoverflow.com/questions/40659212/futurewarning-elementwise-comparison-failed-returning-scalar-but-in-the-futur

Reply:
Ah, okay, thanks.